tests/plugins/widget/manual/copycutintegration.html

```diff
@@ -0,0 +1,35 @@
+<div id="editor1">
+	<p>Adipisicing corporis rem repellendus vel mollitia vero? Consectetur dolores voluptatibus illo ipsam eveniet? Lorem ipsum dolor sit amet.</p>
+	<div data-widget="customwidget">Widget</div>
```

Comment:
This widget is not the best fit for this test, since it's hard to select part of it (e.g on Chrome you can select non or entire widget). Could you make it more complex (e.g. with few elements ore editable part) to make sure that partial selection is possible?